app/backend/approaches/chatreadretrievereadvision.py

```diff
@@ -183,7 +182,7 @@ async def run_until_final_call(
                 ThoughtStep(
                     "Generated search query",
                     query_text,
-                    {"semanticCaptions": use_semantic_captions, "vector_fields": vector_fields},
+                    {"use_semantic_captions": use_semantic_captions, "vector_fields": vector_fields},
```

Comment:
Cosmetic rename, to match the other key. That's what caused many of the snapshot diffs in this PR.